src/main/resources/application.conf

```diff
@@ -26,6 +26,9 @@ lorre: {
   purgeAccountsInterval: 10
   #Used to select how many fees should be averaged together
   numberOfFeesAveraged: 1000
+  #read the tezos net to use from env or default
+  tezos-network: "zeronet"
```

Comment:
Use platforms.tezos to figure out which network to use in the code. "zeronet" is already there

Reply:
great, will check tomorrow

Reply:
you mean that I should look into the existing subtrees in the `platforms.tezos` configuration, and pick the first subtree's root?

what if there is more than one configuration? like 

```
platform.tezos = {
  alphanet = {
  ...
  }
  zeronet = {
  ...
  }
}
```